community-manager-self-care.adoc

```diff
@@ -1,23 +1,644 @@
-
-. Disclaimer
-..Important to note that this isn't medical advice and if you are concerned you are showing signs of burnout, please consult a doctor.
-. Importance of Self-Care
-.. Leaders around the clock
-.. Being human
-.. Types of Self-Care
-. Burnout
-.. Look out for symptoms
-.. Preventing/Treating Burnout
-. Work/Life Balance
-.. Addiction
-.. Maintain boundaries
-.. Sustainability
-. Self-Reflection
-.. Through the looking glass
-.. Tackling Impostor Syndrome
-.. Network of Support
+Authors: Ashley Nicolson <ashley@scotlandopensource.com>
+Updated: 2020-07-09
+
+== Disclaimer
+
+Materials in this chapter are for informational purposes only, not for the purpose of providing medical advice.
+This chapter is not intended as a substitute for professional medical advice, diagnosis or treatment.
+Always seek the advice of your physician or other qualified health care provider with any questions or concerns you may have regarding your health both mental and physical.
+
+== Introduction
+
+Because of their roles (but also, perhaps, because of their natures), community managers often focus (even fixate) on the health and well-being of their community members and teams.
+They understand the challenges and difficulties associated with encouraging stable, safe, and inclusive online communities, places where people demonstrate compassion and empathy.
+But too often, they mistakenly assume they're immune to the same emotionally difficult challenges they strive to help other community members overcome.
+
+What's more, they may even try to _hide_ their vulnerabilities, for fear of being perceived as less capable of performing their roles—or, worse, being rejected by their communities or peer groups group completely.
+This perspective on vulnerability can lead community managers down an isolating road, one that might make them feel as though they aren'table to reach out to seek help when they need it.
+
+And yet, the longer we don't attend to our well being, prolonging treatment for our increasingly negative symptoms, the larger the problem—until, eventually, we are mentally and physically debilitated, suffocated, and denied of the fulfillment and joy our role once provided us.
+Regaining that joy could take months, even years.
+In fact, we may never regain that same level of passion for them again.
+
+To help ourselves as community managers, and in turn help others, we need to learn how to spot the signs of dissatistfaction, disillusionment, and burnout.
+These signs can be subtle and hard to detect, but by identifying and acknowledging these changes in behavior (first and foremost in yourself), you begin applying similar awareness to others in your communities.
+This experience can provide insight to developing and building up social norms and mechanisms for the community to help reduce the occurrence and impact of burnout thus promoting a physically and mentally healthy, more productive community.
+
+_In this chapter, we'll ..._
```

Comment:
I feel like this is a nice opportunity to provide readers with a cursory preview or roadmap of the chapter ahead.

Reply:
Yup, I've updated this. :+1:

Reply:
Awesome! Thanks.